Improve landing page and add overview pages. (#2470)
wso2/docs-is: en/docs/guides/mfa/possession-factor.md

# Possession Factor

Possession factors are hardware entities owned by the user which are used as authentication means to connect to systems. 

> The advantage of possession factors is that remote hackers can't perform any malicious attack to the system as the 

Reviewer: ```suggestion
The advantage of possession factors is that the remote hackers cannot perform any malicious attacks on the system.
```
